docs/design/queue-resource-reservation-design-v2.md

```diff
@@ -0,0 +1,35 @@
+# Volcano Resource Reservation For Queue(v2)
+
+@[Thor-wl](https://github.com/Thor-wl); Feb 28th, 2021
+
+Inspired by design optimization of `target job resource reservation`, our goal is to ensure the SLA of jobs in target
+queues. So just allocate resource for jobs by order SLA.
+
+## Design
+Queues with annotation `maxWaitingTime` indicate that this queue should ensure SLA. Jobs in the queue will be scheduled
+first if its SLA 
```

Comment:
Do you mean "if its SLA arrives."?

Reply:
> Do you mean "if its SLA arrives."?

Sorry for the uncompletion of the doc. I've supplied it.